jwst/resample/gwcs_drizzle.py

```diff
@@ -196,13 +196,10 @@ def add_image(self, insci, inwcs, inwht=None,
         wt_scl = 1.0  # hard-coded for JWST count-rate data
         self.increment_id()
 
-        dodrizzle(insci, inwcs, inwht, self.outwcs,
-                            self.outsci, self.outwht, self.outcon,
-                            expin, in_units, wt_scl,
-                            pscale_ratio=pscale_ratio, uniqid=self.uniqid,
```

Comment:
The 'pscale_ratio' parameter also needs to be removed from the signature of the 'add_image' function (line 117?).

Reply:
Are you - by implication - confirming that `pscale_ratio` should not be used at all in `dodrizzle` (and `add_image`)?

Reply:
It already is not used by the code, making this parameter in the function signature unnecessary...if I didn't overlook any use of this parameter in my review, that is. ;-)

Reply:
Thanks for catching that.  Removed.